### PR TITLE
Add error messages to require() in service contracts

### LINKED ISSUE #1111: Add messages to require()s in service contracts
because that makes debugging easier.

### PR DESCRIPTION
**Pull request by pirapira**
This closes #1111.

### CODE REVIEW

File: raiden_contracts/data/source/services/MonitoringService.sol [hunk 1]
```diff
@@ -222,12 +222,12 @@ contract MonitoringService is Utils {
         (settle_block_number, channel_state) = token_network.getChannelInfo(
             channel_identifier, closing_participant, non_closing_participant
         );
-        require(channel_state == TokenNetwork.ChannelState.Closed);
+        require(channel_state == TokenNetwork.ChannelState.Closed, "not closed");
```
Comment (palango) [inline raiden_contracts/data/source/services/MonitoringService.sol]: maybe *channel not closed*?

File: raiden_contracts/data/source/services/MonitoringService.sol [hunk 2]
```diff
@@ -222,12 +222,12 @@ contract MonitoringService is Utils {
...
 
         // We don't actually know when the channel has been closed. So we'll
         // make a guess so that assumed_close_block >= real_close_block.
         uint256 assumed_settle_timeout = token_network.settlement_timeout_min();
-        require(settle_block_number >= assumed_settle_timeout);
+        require(settle_block_number >= assumed_settle_timeout, "weird settle block number");
```
Comment (palango) [inline raiden_contracts/data/source/services/MonitoringService.sol]: weird -> wrong/to low?

File: raiden_contracts/data/source/services/MonitoringService.sol [hunk 3]
```diff
@@ -65,18 +65,18 @@ contract MonitoringService is Utils {
     )
         public
     {
-        require(_token_address != address(0x0));
-        require(_service_registry_address != address(0x0));
-        require(_udc_address != address(0x0));
-        require(contractExists(_token_address));
-        require(contractExists(_service_registry_address));
-        require(contractExists(_udc_address));
+        require(_token_address != address(0x0), "Token at address zero?");
```
Comment (palango) [inline raiden_contracts/data/source/services/MonitoringService.sol]: why use question marks here? below you don't use them: https://github.com/raiden-network/raiden-contracts/pull/1115/files#diff-75ab51d9425251e3761a502084d14fd7R312

File: raiden_contracts/data/source/services/MonitoringService.sol (hunk 2 above)
Comment (pirapira) [reply raiden_contracts/data/source/services/MonitoringService.sol]: I'll take "too low".

File: raiden_contracts/data/source/services/MonitoringService.sol (hunk 3 above)
Comment (pirapira) [reply raiden_contracts/data/source/services/MonitoringService.sol]: I'll make it consistent.

**Pull request merged**